### ISSUE TITLE
Geo queries polygon tests failing with MongoDB 2.6 version

### ISSUE THREAD

**Issue by fgalan** [telefonicaid/fiware-orion#415]
The following test seem to fail in Debian running MongoDB 2.6 (confirmed in @LeandroGuillen and my environment):

```
[  FAILED  ] mongoQueryContextGeoRequest.queryGeoPolygonOut1
[  FAILED  ] mongoQueryContextGeoRequest.queryGeoPolygonOut2
```

Fermín is using MongoDB 2.6.1 in particular.

**Comment by fgalan** [comment]
The problem has been identified. A question has been posted in SOF in order to get more insight in the best of solving it: http://stackoverflow.com/questions/29388981/different-semantics-in-not-geowithin-with-polygon-geometries-between-mongodb-2

**Comment by fgalan** [comment]
Implemented in PR #871. However, this issue cannot be closed until a script to migrate existing DBs chanding `location.coord` from plain vectors to GJSON "Point" gets developed.

**Comment by fgalan** [comment]
At the end, the script was added as part of PR #781
